package/yast2-storage-ng.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Mon Jan 15 08:22:32 UTC 2018 - jlopez@suse.com
+
+- Added sanity checks for partitioning setup.
```

Comment:
NP: I would mention the expert partitioner here (and maybe also the new control.xml format somehow).